packages/engine/vscode/src/lib/editor.ts

```diff
@@ -1 +1,93 @@
-// Look at this : https://microsoft.github.io/language-server-protocol/specifications/specification-current/
\ No newline at end of file
+import { CommandPlugin, PluginOptions } from "@remixproject/engine-vscode";
+import { Annotation, HighlightPosition } from '@remixproject/plugin-api';
+import { window, Range, TextEditorDecorationType, Position, languages, DiagnosticCollection, Diagnostic, Uri, DiagnosticSeverity, TextEditor } from "vscode";
+
+const profile = {
```

Comment:
You should be able to import it from @remixproject/plugin-api too. Check how I did it in filemanager